dashboard/test/controllers/levels_controller_test.rb

```diff
@@ -3,6 +3,8 @@
 class LevelsControllerTest < ActionController::TestCase
   include Devise::Test::ControllerHelpers
 
+  default_update_blocks_params = nil
```

Comment:
Weird. I played around with it and confirmed that it worked. But not for the reason it seems. This has to do with the closures generated for the blocks sent to the `test` command when the class is defined. 

See this example:
```ruby
class C
  temp_var = 'value'

  define_method :with_closure do
    puts "with closure: #{defined?(temp_var)}"
  end

  def without_closure
    puts "without closure: #{defined?(temp_var)}"
  end

  define_method :nested do
    print "nested, "
    without_closure
  end
end

c = C.new
c.with_closure
c.without_closure
c.nested
```
The output is:
```
with closure: local-variable
without closure:
nested, without closure:
```

It shows up in the closure as if it were a local variable, but doesn't show up anywhere else.
That is only AFAICT because the blocks were defined in the same context. That variable is only in scope while the class is loaded, not accessible anywhere else later.

Similarly,
```ruby
class C
  @outside = nil

  def initialize
    @inside = nil    
  end

  def m
    puts "outside: #{defined?(@outside)}, inside: #{defined?(@inside)}"
  end
end

C.new.m
```
outputs
```
outside: , inside: instance-variable
```
The instance variable only persists when it's defined in a method. The one defined in the class body is lost.